decentraland/unity-renderer
# Fix: temporary fix for avatar editor invisible for guests

## What does this PR change?

Fix #3018

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/avatar-editor-guest
2. Access as guest user
3. Verify that the regular avatar creation flow and the explore are working as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
